service/src/main/java/skills/services/SkillEventAdminService.groovy

```diff
@@ -171,6 +174,19 @@ class SkillEventAdminService {
         return results
     }
 
+    @Profile
+    @Transactional
+    RequestResult bulkDeleteSkillEventsForUser(String projectId, String userId) {
+        RequestResult res = new RequestResult()
+        performedSkillRepository.deleteAllByUserIdAndProjectId(userId, projectId)
+        userEventService.removeAllEvents(projectId, userId)
+        achievedLevelRepo.deleteAllByProjectIdAndUserId(projectId, userId)
+        userPointsRepo.deleteAllByProjectIdAndUserId(projectId, userId)
+        skillApprovalRepo.deleteAllByProjectIdAndUserId(projectId, userId)
+        res.success = true
+        return res
```

Comment:
really tiny thing but could save few lines of code with ``return RequestResult.success()``